src/aiida/cmdline/commands/cmd_process.py

```diff
@@ -474,3 +532,94 @@ def process_repair(manager, broker, dry_run):
         if pid not in set_process_tasks:
             process_controller.continue_process(pid)
             echo.echo_report(f'Revived process `{pid}`')
+
+
+@verdi_process.command('dump')
+@arguments.PROCESS()
+@options.PATH()
+@options.NO_NODE_INPUTS()
+@options.INCLUDE_ATTRIBUTES()
+@options.INCLUDE_EXTRAS()
+@options.USE_PRESUBMIT()
+@options.OVERWRITE()
+def dump(
+    process,
+    path,
+    no_node_inputs,
+    include_attributes,
+    include_extras,
+    use_presubmit,
+    overwrite,
+) -> None:
+    """Dump files involved in the execution of a process.
+
+    Child simulations/workflows (also called `CalcJob`s and `WorkChain`s in AiiDA jargon) run by the parent workflow are
+    contained in the directory tree as sub-folders and are sorted by their creation time. The directory tree thus
+    mirrors the logical execution of the workflow, which can also be queried by running `verdi process status
+    <pk>` on the command line.
+
+    By default, input and output files of each simulation can be found in the corresponding "raw_inputs" and
+    "raw_outputs" directories (the former also contains the hidden ".aiida" folder with machine-readable job execution
+    settings). Additional input files (depending on the type of calculation) are placed in the "node_inputs".
+
+    When using the `--use-presubmit` command line option, the folder created for each individual simulation should, in
+    principle, allow for direct resubmission, as it mirrors the (remote) folder that was created by AiiDA to execute the
+    job. However, this option requires the relevant AiiDA plugin to be installed, so it is disabled by default. Also
+    note that intermediate files might be missing, so for a multi-step workflow, each step would still have to be run
+    separately.
+
+    Lastly, every folder also contains a hidden, human-readable `.aiida_node_metadata.yaml` file with the relevant AiiDA
+    node data for further inspection.
+    """
+
+    from aiida.orm import CalcJobNode
+    from aiida.tools.dumping.processes import (
+        ProcessNodeYamlDumper,
+        calcjob_dump,
+        generate_default_dump_path,
+        make_dump_readme,
+        workchain_dump,
+    )
+
+    # Generate default parent folder
+    if str(path) == '.':
+        output_path = generate_default_dump_path(process_node=process)
+
+    # Instantiate YamlDumper
+    processnode_dumper = ProcessNodeYamlDumper(include_attributes=include_attributes, include_extras=include_extras)
+
+    # Allow the command to be run with `CalcJob`s and `WorkChain`s
+    if isinstance(process, CalcJobNode):
+        echo.echo_warning('Command called on `CalcJob`. Will dump, but you can also use `verdi calcjob dump` instead.')
```

Comment:
Yeah, it is a bit weird indeed. Eventually, do we even still want to make `verdi calcjob dump` available? Iirc you've
been advocating against it, and I am also gradually leaning more towards that.

Reply:
I don't really see the point to be honest. Since `verdi process dump` would work for any process node, with an identical interface, I don't see what adding the other commands accomplishes other than cause potential confusion.

Reply:
I'm on board with removing it (and already did so in my latest commit :)